Mathlib/NumberTheory/LSeries/Basic.lean

```diff
@@ -143,6 +143,26 @@ lemma LSeriesSummable_congr {f g : ℕ → ℂ} (s : ℂ) (h : ∀ n ≠ 0, f n
     LSeriesSummable f s ↔ LSeriesSummable g s :=
   summable_congr <| term_congr h s
 
+open Filter in
+/-- If `f` and `g` agree on large `n : ℕ`, then the `LSeries` of `f` converges at `s`
+if and only if that of `g` does. -/
+lemma LSeriesSummable_congr' {f g : ℕ → ℂ} (s : ℂ) (h : f =ᶠ[atTop] g) :
```

Comment:
The usual practice for `congr` lemmas is, I think, to have a one-way implication set up for dot-notation:
````
lemma LSeriesSummable.congr' {f g : ℕ → ℂ} {s : ℂ} (hf : LSeriesSummable f s) (hfg : f =ᶠ[atTop] g) : LSeriesSummable g s := sorry
````
and then the `iff` version outside the namespace. Since you are already proving the one-way implication inside your proof anyway, this just amounts to giving it a name.

Reply:
OK, done.